README.md

```diff
@@ -271,6 +271,20 @@ In order to add a new workspace, one should create a new directory under `packag
 
 Suggested way to add a new dependency to an existing workspace is to run `yarn workspace <workspace name> add <package name>` instead of just `yarn add <package name>`. For example, run `yarn workspace cbioportal-frontend add lodash` instead of `yarn add lodash`. Similarly to remove a package run `yarn workspace <workspace name> remove <package name>`.
 
+### Tips for dependency management
+
+Since now all the packages live under a single repository it might be confusing how to import different modules into the file you're working on.
+
+If you are working on `cbioportal-frontend` repository feel free to import any module from any package under `packages`. When importing from internal packages always use import statements like `import {something} from 'cbioportal-frontend-commons'`. Never do under any circumstances something like `import {something} from ../../packages/cbioportal-frontend-commons/src/lib/someLib` or `import {something} from ../../packages/react-mutation-mapper/src/component/someComponent`. Treat internal packages like any other npm package you would import.
```

Comment:
might be good to have a circle ci check or github action for this at some point, could be as simple as

```
git grep -q "../packages/" src/** && exit 1
```

Reply:
but maybe this is already caught by the build process itself?